dags/multipod/configs/maxtext_gce_config.py

```diff
@@ -75,7 +75,82 @@ def get_maxtext_nightly_config(
       set_up_cmds=set_up_cmds,
       run_model_cmds=run_model_cmds,
       time_out_in_min=time_out_in_min,
-      task_owner=test_owner.Tony_C,
+      task_owner=test_owner.TONY_C,
+      num_slices=num_slices,
+  )
+
+  return task.TpuQueuedResourceTask(
+      task_test_config=job_test_config,
+      task_gcp_config=job_gcp_config,
+  )
+
+
+def get_maxtext_stable_decode_config(
+    tpu_version: TpuVersion,
+    tpu_cores: int,
+    tpu_zone: str,
+    time_out_in_min: int,
+    test_name: str,
+    model_ckpt_path: str,
+    model_ckpt_folder: str,
+    model_name: str,
+    expected_decode_result: str,
+    project_name: str = PROJECT_NAME,
+    runtime_version: str = RUNTIME_IMAGE,
+    network: str = "default",
+    subnetwork: str = "default",
+    is_tpu_reserved: bool = True,
+    num_slices: int = 1,
+) -> task.TpuQueuedResourceTask:
+  job_gcp_config = gcp_config.GCPConfig(
+      project_name=project_name,
+      zone=tpu_zone,
+      dataset_name=metric_config.DatasetOption.XLML_DATASET,
+  )
+
+  set_up_cmds = common.setup_maxtext() + (
+      "pip3 install torch",
+      "pip3 install --no-cache-dir -U crcmod",
+  )
+
+  # download the checkpoint, convert it, and run script
+  current_datetime = datetime.datetime.now().strftime("%Y-%m-%d-%H-%M-%S")
+  parent_path = f"{gcs_bucket.XLML_OUTPUT_DIR}/maxtext/stable/{model_name}"
+  export_vars = (
+      "export M_ENABLE_CHECKPOINTING=true && export M_DATASET_PATH=gs://maxtext-dataset &&"
```

Comment:
I actually think all of this logic should be in MaxText, not in XLML. The idea is that MaxText holds the "test". What configurations run would be controlled by XLML.

Reply:
Yes, this could also work. The configuration needed by XL ML is script to setup and run training/inference/whatever. I could just write a helper function to call a bash script from MaxText repo.

Reply:
Happy to discuss more details offline! Once we are aligned, I can update the test accordingly. Thank you!

Reply:
Discussed offline.